lib/mvos/service/firebase_service.dart

```diff
@@ -31,7 +48,8 @@ class FirebaseService implements Disposable {
   void _initFirebase() async {
     FirebaseAuth.instance.onAuthStateChanged.listen((firebaseUser) {
       logger.info('auth state changed $firebaseUser');
-      userE$.add(UserE(uid: firebaseUser?.uid));
+      userE$.add(
```

Comment:
nakon sto se promijeni auth state, pozoves iz databasea query za usera sa tim uid-om.
I onda mozes dodati u UserE i ostale parametre -> name surname ...

Reply:
Do we need UserE and FirebaseUserE ?

Reply:
Maybe I made a mistake, I add name,surname.. etc in the FirebaseUserE ?

Reply:
ovo je ok za sad. naminge cemo kasnije u jednom commitu